net/tcp/tcp_conn.c

```diff
@@ -538,22 +570,10 @@ static inline int tcp_ipv6_bind(FAR struct tcp_conn_s *conn,
 
 void tcp_initialize(void)
 {
-  int i;
-
   /* Initialize the queues */
 
   dq_init(&g_free_tcp_connections);
   dq_init(&g_active_tcp_connections);
-
-  /* Now initialize each connection structure */
-
-  for (i = 0; i < CONFIG_NET_TCP_CONNS; i++)
-    {
-      /* Mark the connection closed and move it to the free list */
-
-      g_tcp_connections[i].tcpstateflags = TCP_CLOSED;
```

Comment:
Where is this functionality now?

Reply:
line 548 - 550
https://github.com/apache/incubator-nuttx/pull/5104/commits/6038a35bc5c57d17c6ab277906d5c3e696a43c27#diff-fd66b1e5b4be7133f600f2a9089d2415165ef1fd8ff957e8aef261b119f7394eR548-R550